tyk-stack/README.md

```diff
@@ -35,13 +37,10 @@ helm repo update
 helm show values tyk-helm/tyk-stack > values-stack.yaml --devel
 ```
 
-*If you use the Bitnami chart for Redis installation, the DNS name of your Redis as set by Bitnami is `tyk-redis-master.tyk.svc.cluster.local:6379`.
-
-You can update them in your local `values-stack.yaml` file under `global.redis.addr` and `global.redis.pass`.
-
-Alternatively, you can use `--set` flag to set it in Tyk installation. For example `--set global.redis.pass=$REDIS_PASSWORD`
-
-*All the values above are just examples, please input the values specific for your deployment.
+At a minimum, modify values-stack.yaml for the following settings:
+1. [Set Redis connection details](#set-redis-connection-details-required)
+2. [Set Mongo or PostgresSQL connection details](#set-mongo-or-postgressql-connection-details-required)
+3. [Dashboard License](#dashboard-license)
```

Comment:
Additionally, would it be good to include an additional conditional step for the Developer Portal license, in case users opt to use it?

Reply:
👍 will add to it